doitlive.py

```diff
@@ -181,6 +184,18 @@ class TTY(object):
     DIM = ANSICode(dim=True)
 
 
+def echo(message=None, file=None, nl=True, err=False, color=None, carriage_return=False):
```

Comment:
I need access to the `nl` parameter though, so it seemed easier to explicitly specify them instead of manipulation kwargs.